internal/connrt/connection_printer.go

```diff
@@ -0,0 +1,54 @@
+package connrt
+
+import (
+	"github.com/gookit/event"
+	"github.com/kbence/conndetect/internal/connlib"
+	"github.com/kbence/conndetect/internal/utils"
+)
+
+type ConnectionPrinter struct {
+	Node
+
+	printer utils.Printer
+	time    utils.Time
+}
+
+func NewConnectionPrinter(eventManager *event.Manager) *ConnectionPrinter {
+	printer := &ConnectionPrinter{
+		Node: Node{eventManager: eventManager},
+
+		printer: utils.NewPrinter(),
+		time:    utils.NewTime(),
+	}
+
+	event.On("newConnection", event.ListenerFunc(printer.Handle))
```

Comment:
We would normally encourage having `newConnection` as a constant somewhere, to avoid hard to notice typos and produce compile time failures on a typo.

Reply:
Yeah, I was thinking about it but then I didn't fix it immediately. But it's fixed now, thanks for the feedback!